src/@orb.yml

```diff
@@ -1,86 +1,8 @@
 version: 2.1
 
 description: >
-  Simplify common tasks for building and testing Node projects.
-  Source: https://github.com/circleci-public/node-orb
+  Easily install NodeJS or its package managers. Best of all, install your node_modules with caching enabled by default.
 
-examples:
-  quickstart-tests:
-    description: >
-      A simple install dependencies and test job that relies on npm conventions
-      and will cache dependencies between builds.
-    usage:
-      version: 2.1
-      orbs:
-        node: circleci/node@x.y.z
-      workflows:
-        build:
-          jobs:
-            - node/test
-
-  quickstart-running-commands:
-    description: >
-      If you want to run a different command after installing dependencies
-      you can do that too.
-    usage:
-      version: 2.1
-      orbs:
-        node: circleci/node@x.y.z
-      workflows:
-        build:
-          jobs:
-            - node/run:
-                tag: lts
-                command: npm run generate-docs
-
-  installing-dependencies:
-    description: >
-      You can also use the dependency install and caching commands directly in
-      your own jobs.
-    usage:
-      version: 2.1
-      orbs:
-        node: circleci/node@x.y.z
-      job:
-        build:
-          - run: echo "Install dependencies using 'npm ci' with a cache"
-          - node/npm-ci
-
-          - run: echo "Or use different commands with the cache"
-          - node/with-cache:
-              cache-key: yarn.lock
-              steps:
-                - run: yarn install --frozen-lockfile
-
-  quickstart-tests-with-results:
-    description: >
-      If your test runner is configured to produce Junit XML, we can save and
-      parse the details of failed tests to display in the UI.
-    usage:
-      version: 2.1
-      orbs:
-        node: circleci/node@x.y.z
-      workflows:
-        build:
-          jobs:
-            - node/test:
-                tag: lts
-                test-results-path: /tmp/test-results
-
-  quickstart-tests-with-lint-and-codecov:
-    description: >
-      You can also specify extra commands to run before or after tests.
-    usage:
-      version: 2.1
-      orbs:
-        node: circleci/node@x.y.z
-        codecov: codecov/codecov@1.0.5
-      workflows:
-        build:
-          jobs:
-            - node/test:
-                tag: lts
-                before-tests:
-                  - run: npm run lint
-                after-tests:
-                  - codecov/upload
+display:
+  source_url: https://github.com/circleci-public/node-orb
+  home_url: https://nodejs.org/
```

Comment:
@lokst has been using the `home_url` key for our own orbs to point to Docs that include examples of the orb. Would that be more useful than pointing to the Node.js website?

Reply:
Ill be talking with the sample project team soon and maybe we could potentially create a blog post around this but Im not sure what else we would link to for a home page for this orb. Open to suggestions.

Reply:
Yeah if there's nothing available now, then this URL is good. Just something to think about so maybe we can change this in the future when we have a doc/sample project.